src/types/Users.ts

```diff
@@ -0,0 +1,6 @@
+export interface Users {
```

Comment:
```suggestion
export interface User {
```
It's only one user, not users